.buildkite/scripts/run_models.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+
+
```

Comment:
In order to avoid false positive buildkite runs, we need to add `set -euo pipefail` here. See how this was done in our Ad Reporting package.

https://github.com/fivetran/dbt_ad_reporting/blob/8c30e5d8fef1c5844eb63312990c2a55d9c3eecc/.buildkite/scripts/run_models.sh#L3

Reply:
Done.